sdk/servicebus/service-bus/test/browserified/renewLock.spec.ts

```diff
@@ -0,0 +1,364 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import chai from "chai";
+const should = chai.should();
+import chaiAsPromised from "chai-as-promised";
+import * as dotenv from "dotenv";
+dotenv.config();
+chai.use(chaiAsPromised);
+import {
+  ServiceBusClient,
+  QueueClient,
+  TopicClient,
+  SubscriptionClient,
+  OnMessage,
+  ServiceBusMessage,
+  MessagingError,
+  OnError,
+  ReceiveMode
+} from "../../src";
+import { delay } from "rhea-promise";
+import { purge, getSenderReceiverClients, TestClientType, TestMessage } from "./testUtils";
+
+let ns: ServiceBusClient;
+let senderClient: QueueClient | TopicClient;
+let receiverClient: QueueClient | SubscriptionClient;
+
+async function beforeEachTest(
+  senderType: TestClientType,
+  receiverType: TestClientType
+): Promise<void> {
+  // @ts-ignore
+  if (!window.__env__["SERVICEBUS_CONNECTION_STRING"]) {
+    throw new Error(
+      "Define SERVICEBUS_CONNECTION_STRING in your environment before running integration tests."
+    );
+  }
+
+  // @ts-ignore
+  ns = ServiceBusClient.createFromConnectionString(window.__env__["SERVICEBUS_CONNECTION_STRING"]);
+  const clients = await getSenderReceiverClients(ns, senderType, receiverType);
+  senderClient = clients.senderClient;
+  receiverClient = clients.receiverClient;
+
+  await purge(receiverClient);
+  const peekedMsgs = await receiverClient.peek();
+  const receiverEntityType = receiverClient instanceof QueueClient ? "queue" : "topic";
+  if (peekedMsgs.length) {
+    chai.assert.fail(`Please use an empty ${receiverEntityType} for integration testing`);
+  }
+}
+
+async function afterEachTest(): Promise<void> {
+  await ns.close();
+}
+
+describe("Unpartitioned Queue - Lock Renewal", function(): void {
```

Comment:
I would say lets keep just the below 3

- Streaming Receiver: renewLock() resets lock duration each time.
- Streaming Receiver: complete() after lock expiry with auto-renewal disabled throws error
- Streaming Receiver: lock will not expire until configured time

These cover the `renewMessageLock()` and the auto lock renewal feature.
Other tests would be covering the same code paths as above as far as locks are concerned